packages/admin/src/utils/QuestionPageGetServerSideProps.tsx

```diff
@@ -100,20 +100,22 @@ async function postPagesResult<
   req,
   res,
   handleRequest,
-  fetchPageDataErrorHandler,
```

Comment:
As discussed can get rid of this prop, as fetchPageData can return redirects and in the one place we need some special try catch behaviour we can just shift it into that function